src/main/java/com/shivankshi/emscrud/rest/EmployeeRestExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.shivankshi.emscrud.rest;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class EmployeeRestExceptionHandler {
+
+	@ExceptionHandler
+	public ResponseEntity<EmployeeErrorResponse> handleException(EmployeeNotFoundException exc) {
+		EmployeeErrorResponse error = new EmployeeErrorResponse();
+		error.setStatus(HttpStatus.NOT_FOUND.value());
+		error.setMessage(exc.getMessage());
+		error.setTimeStamp(System.currentTimeMillis());
+
+		return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
+
+	}
+
+	@ExceptionHandler
+	public ResponseEntity<EmployeeErrorResponse> handleException(Exception exc) {
```

Comment:
ControllerAdvice is defaulting to base exception.

Reply:
Code is working fine.
Error was coming due to executing the old branch which was not updated.